server/utils/httpCodes.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    Success: 200,
+    Created: 201,
+    SuccessNoContent: 204,
+    BadRequest: 400,
+    Unauthenticated: 401,
+    Unauthorized: 403,
+    NotFound: 404,
+    Unprocessable: 422,
+    ServerError: 500
+}
```

Comment:
I like the motivation behind extracting constants but I'm not sure this use case makes sense. HTTP codes are static across all applications so you can reliably hard code them because their meaning won't change